style-guides/scss.md

```diff
@@ -0,0 +1,123 @@
+<script>{
+	"title": "Chassis SCSS Style Guide"
+}</script>
+
+This page outlines the style guide for SCSS for the jQuery Chassis project.
+
+## Comments
+
+SCSS comments should follow the valid CSS comment guidelines outlined in the jQuery contribute CSS style guide.
+
+```scss
+/* Good single line comment */
+
+/*
+* Good example of a multi-line comment.
+* If your comment takes up multiple lines, please do this.
+*/
+```
+
+## Cascading
+
+Avoid cascading as often as possible and follow the `BEM` naming conventions. If cascading is necessary avoid cascading selectors that are more than 3 selectors long.
+
+```scss
+/* 
+* Example without BEM using cascading 
+* (Please avoid this)
+*/
+.login {
+	.submit {
+		.disabled {
+
+		}
+	}
+}
+
+/* Good example using BEM */
+.login {
+	/* Block */
+	&__submit {
+		/* Element */
+		&--disabled {
+			/* Modifier */
+		}
+	}
+}
+```
+
+## Rule Order
+
+To keep CSS consistant and easy to navigate please use the following order for each rule:
+
+1. scoped variables
+2. @extend
+3. @include
+4. regular styles
+	- Sort declarations based on type such as padding/margin, position/top/right/bottom/left etc.
+5. pseudo classes and elements
+6. media queries
```

Comment:
From what i'v always seen it seems like general convention is to put media queries last